pkg/deployments/tasks/20220415-primary-issue-pool/test/test.fork.ts

```diff
@@ -48,33 +50,53 @@ describeForkTest('PrimaryPoolFactory', 'goerli', 8586768, function () {
-        'Authorizer'
+            'Authorizer'
         );
 
-        vcusd = await task.instanceAt('IERC20', VCUSD);
+        ``
+
         usdc = await task.instanceAt('IERC20', USDC);
+        vcusd = await task.instanceAt('IERC20', VCUSD);
         usdt = await task.instanceAt('IERC20', USDT);
+
+        await usdc.connect(whale).approve(vault.address, MAX_UINT256);
+        await vcusd.connect(whale).approve(vault.address, MAX_UINT256);
+        await usdt.connect(whale).approve(vault.address, MAX_UINT256);
+
+
     });
 
     describe('create and swap', () => {
         let pool: Contract;
         let poolId: string;
 
-        it('deploy a primary issue pool', async () => {   
-            const tx = await factory.create({name: 'Verified Pool Token', symbol: 'VPT', security: usdc.address, currency: vcusd.address, minimumPrice: minimumPrice, minimumOrderSize: minimumOrderSize, maxAmountsIn: maxSecurityOffered, swapFeePercentage: swapFeePercentage, cutOffTime: issueCutoffTime, offeringDocs: offeringDocs});
-            const event = expectEvent.inReceipt(await tx.wait(), 'PoolCreated');
+        it('deploy a primary issue pool', async () => {
+            try {
+                const tx = await factory.create({
+                    name: 'Verified Pool Token', symbol: 'VPT', security: usdc.address, currency: vcusd.address, minimumPrice: minimumPrice, minimumOrderSize: minimumOrderSize, maxAmountsIn: maxSecurityOffered, swapFeePercentage: swapFeePercentage, cutOffTime: issueCutoffTime, offeringDocs: offeringDocs, initialBalances: initialBalances,
+                });
+                console.log(await tx.wait());
+                const event = await expectEvent.inReceipt(await tx.wait(), 'PoolCreated');
 
-            pool = await task.instanceAt('PrimaryIssuePool', event.args.pool);
-            expect(await factory.isPoolFromFactory(pool.address)).to.be.true;
+                const pool = await task.instanceAt('PrimaryIssuePool', event.args.pool);
+                expect(await factory.isPoolFromFactory(pool.address)).to.be.true;
 
-            poolId = await pool.getPoolId();
-            const [registeredAddress] = await vault.getPool(poolId);
-            expect(registeredAddress).to.equal(pool.address);
+                poolId = await pool.getPoolId();
+                const [registeredAddress] = await vault.getPool(poolId);
+                expect(registeredAddress).to.equal(pool.address);
+            } catch (error: any) {
+                console.error('Error deploying primary issue pool:', error.message);
+                throw error;
+            }
         });
 
-        it('initialize the pool', async () => {
-            await vcusd.connect(whale).approve(vault.address, MAX_UINT256);
+
+
+        it('Initialize the pool', async () => {
             await usdc.connect(whale).approve(vault.address, MAX_UINT256);
+            await usdc.connect(whale).approve(vault.address, MAX_UINT256);
```

Comment:
@vinaykumar0103 should this approval be for VCUSD ? We already have given approval for USDC in the above line.